testutil/testclient/testqueryclients/servicequerier.go

```diff
@@ -0,0 +1,61 @@
+package testqueryclients
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/golang/mock/gomock"
+
+	"github.com/pokt-network/poktroll/testutil/mockclient"
+	sharedtypes "github.com/pokt-network/poktroll/x/shared/types"
```

Comment:
It also looks like the `prooftypes` import is missing:

```suggestion
	prooftypes "github.com/pokt-network/poktroll/x/proof/types"
	sharedtypes "github.com/pokt-network/poktroll/x/shared/types"
```